pkg/utils/authcheck/probechecker_test.go

```diff
@@ -90,3 +85,16 @@ func TestProbeCheckResult(t *testing.T) {
 		})
 	}
 }
+
+func getFreePort() (int, error) {
```

Comment:
Pass in `t` and have it call `t.Fatal` inside the function.

```suggestion
func getFreePort(t *testing.T) int {
```